src/plugins/auto/auto_executable_network.cpp

```diff
@@ -172,6 +172,9 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
         }
         return decltype(ov::available_devices)::value_type {exeDevices};
     } else if (name == ov::model_name) {
+        std::lock_guard<std::mutex> lock(_autoSContext->_confMutex);
+        if (_autoSchedule->_loadContext[CPU].isEnabled && _autoSchedule->_loadContext[CPU].isAlready)
```

Comment:
Will merge this PR as hot fix.

Please use `execution_device` property directly to get property from underlying device in following PR.